torch/fx/graph.py

```diff
@@ -236,10 +242,22 @@ def python_code(self, root_module: str) -> Tuple[str, str, List[str]]:
                 assert isinstance(node.target, str)
                 body.append(f'{node.name} = {_format_target(root_module, node.target)}\n')
                 continue
+            elif node.op == 'output':
+                if result_value:
```

Comment:
It'd be conceptually simpler to just have output insert a `return x` into the body and delete the return formatting below. I wouldn't worry about enforcing single return semantics here, that is what the lint pass should be doing.